# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ppl/ppl-0.11.2-replace-pkglib_DATA.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix regression after automake update from 1.11.1 -> 1.11.6

diff -ruN ppl-0.11.2.orig/interfaces/Prolog/Ciao/Makefile.am ppl-0.11.2/interfaces/Prolog/Ciao/Makefile.am
--- ppl-0.11.2.orig/interfaces/Prolog/Ciao/Makefile.am	2011-02-27 10:07:47.000000000 +0100
+++ ppl-0.11.2/interfaces/Prolog/Ciao/Makefile.am	2013-08-29 00:21:29.941177494 +0200
@@ -170,7 +170,7 @@
 -module \
 -avoid-version
 
-pkglib_DATA = ppl_ciao.po
+pkgdata_DATA = ppl_ciao.po
 
 ppl_ciao.pl: $(interface_generator_dependencies)
 	$(M4) --prefix-builtin -I../.. -I$(srcdir)/.. -I$(srcdir)/../.. \
diff -ruN ppl-0.11.2.orig/interfaces/Prolog/GNU/Makefile.am ppl-0.11.2/interfaces/Prolog/GNU/Makefile.am
--- ppl-0.11.2.orig/interfaces/Prolog/GNU/Makefile.am	2011-02-27 10:07:47.000000000 +0100
+++ ppl-0.11.2/interfaces/Prolog/GNU/Makefile.am	2013-08-29 00:21:29.945177774 +0200
@@ -169,7 +169,7 @@
 	$(M4) --prefix-builtin -I../.. -I$(srcdir)/.. -I$(srcdir)/../.. \
            $(srcdir)/ppl_interface_generator_gprolog_pl.m4 > $@
 
-pkglib_DATA = ppl_gprolog.pl
+pkgdata_DATA = ppl_gprolog.pl
 
 bin_PROGRAMS = ppl_gprolog
 
diff -ruN ppl-0.11.2.orig/interfaces/Prolog/SICStus/Makefile.am ppl-0.11.2/interfaces/Prolog/SICStus/Makefile.am
--- ppl-0.11.2.orig/interfaces/Prolog/SICStus/Makefile.am	2011-02-27 10:07:47.000000000 +0100
+++ ppl-0.11.2/interfaces/Prolog/SICStus/Makefile.am	2013-08-29 00:21:29.941177494 +0200
@@ -69,7 +69,7 @@
 
 if ENABLE_SHARED
 
-pkglib_DATA = ppl_sicstus.pl ppl_sicstus.s.o
+pkgdata_DATA = ppl_sicstus.pl ppl_sicstus.s.o
 
 ppl_sicstus.so: ppl_sicstus.pl $(top_builddir)/src/libppl.la $(WATCHDOG_LIBRARY) libppl_sicstus.la
 	$(LIBTOOL) --mode=link --tag=CXX \
@@ -87,7 +87,7 @@
 
 else !ENABLE_SHARED
 
-pkglib_DATA = ppl_sicstus.pl
+pkgdata_DATA = ppl_sicstus.pl
 
 endif !ENABLE_SHARED
 
diff -ruN ppl-0.11.2.orig/interfaces/Prolog/SWI/Makefile.am ppl-0.11.2/interfaces/Prolog/SWI/Makefile.am
--- ppl-0.11.2.orig/interfaces/Prolog/SWI/Makefile.am	2011-02-27 10:07:47.000000000 +0100
+++ ppl-0.11.2/interfaces/Prolog/SWI/Makefile.am	2013-08-29 00:21:29.929176638 +0200
@@ -189,7 +189,7 @@
 bin_PROGRAMS = ppl_pl
 ppl_pl_SOURCES = ppl_pl.cc
 
-pkglib_DATA = ppl_swiprolog.pl
+pkgdata_DATA = ppl_swiprolog.pl
 
 CLPQ_TESTS_NONSTRICT = \
 ack.clpq \
diff -ruN ppl-0.11.2.orig/interfaces/Prolog/XSB/Makefile.am ppl-0.11.2/interfaces/Prolog/XSB/Makefile.am
--- ppl-0.11.2.orig/interfaces/Prolog/XSB/Makefile.am	2011-02-27 10:07:47.000000000 +0100
+++ ppl-0.11.2/interfaces/Prolog/XSB/Makefile.am	2013-08-29 00:21:29.945177774 +0200
@@ -188,7 +188,7 @@
 -module \
 -avoid-version
 
-pkglib_DATA = ppl_xsb.xwam
+pkgdata_DATA = ppl_xsb.xwam
 
 ppl_xsb.xwam: ppl_xsb.H ppl_xsb.cc libppl_xsb.la
 	cp -f ppl_xsb.cc ppl_xsb.c
diff -ruN ppl-0.11.2.orig/interfaces/Prolog/YAP/Makefile.am ppl-0.11.2/interfaces/Prolog/YAP/Makefile.am
--- ppl-0.11.2.orig/interfaces/Prolog/YAP/Makefile.am	2011-02-27 10:07:47.000000000 +0100
+++ ppl-0.11.2/interfaces/Prolog/YAP/Makefile.am	2013-08-29 00:21:29.945177774 +0200
@@ -179,7 +179,7 @@
 -module \
 -avoid-version
 
-pkglib_DATA = ppl_yap.pl
+pkgdata_DATA = ppl_yap.pl
 
 CLPQ_TESTS_NONSTRICT = \
 ack.clpq \
